### PR TITLE
TVB-2114 PyPi support

### PR DESCRIPTION
**Pull request by danvladpop**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -53,16 +53,9 @@
...
-required to perform whole brain simulation. It is a work in 
-progress, and a subject of on-going research efforts. Please refer
-to the following article for more information: 
-
-http://www.frontiersin.org/Journal/10.3389/fninf.2013.00010/abstract
-
-"""
+    download_url = 'https://github.com/the-virtual-brain/tvb-library',
+    keywords = 'brain simulator neuroscience human animal neuronal dynamics models delay',
+    long_description=open('README.md').read()
```
Comment (maedoc) [inline setup.py]: 1. please don't be sloppy.  read the file properly, above the call to setup,
```python
with open('README.md') as fd:
    long_description = fd.read()
```
and then use the value in the call to setup
```
    long_description=long_description,
```
2. no spaces around keyword args, e.g. `a=3`, not `a = 3`

Comment (maedoc) [reply setup.py]: @danvladpop thanks!

**Comment by liadomide** [comment]
On pypi test currently appears:

>   Home Page: https://github.com/the-virtual-brain/scientific_library  

Can we make it be:  

> Home Page: http://www.thevirtualbrain.org ?

**Comment by liadomide** [comment]
I think that files _requirements.txt_ and _MANIFEST.in_ in the root of this repo are now deprecated, and could be remove. I recommend testing without them to deploy on test.pypi.

**Pull request merged**